pkg/coreapi/gql.schema.graphql

```diff
@@ -130,6 +130,7 @@ type Function {
   name: String!
   slug: String!
   config: String!
+  configuration: FunctionConfiguration
```

Comment:
```suggestion
  configuration: FunctionConfiguration!
```

When would a function not have a config?

Reply:
makes sense, I'll backport to https://github.com/inngest/monorepo/blob/dcaef52cdb93e28d0683af80e4a4d59c96f60147/api/schema.graphql#L838 as well